### ISSUE TITLE
the library SEAL

### ISSUE THREAD

**Issue by cdmc-zrz** [sintefmath/implicit-recon#3]
How did you install the third party library SEAL? This library keeps failing to install for me

**Comment by oliverjdb** [comment]
It seems the installation method for SEAL has changed. Have you tried `pip install spline-algorithm-library`?

**Comment by cdmc-zrz** [comment]
Thanks, seal works, but val loss=NAN occurs during training, is the learning rate too high?




------------------&nbsp;Original&nbsp;------------------
From: &nbsp;"Oliver ***@***.***&gt;;
Send time:&nbsp;Monday, Mar 28, 2022 2:08 AM
***@***.***&gt;; 
***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [sintefmath/implicit-recon] the library SEAL (Issue #3)





 
It seems the installation method for SEAL has changed have you tried pip install spline-algorithm-library?
